iron-list.html

```diff
@@ -1718,6 +1722,7 @@
       this._focusedItem = null;
       this._focusedVirtualIndex = -1;
       this._focusedPhysicalIndex = -1;
+      document.activeElement && document.activeElement.blur();
```

Comment:
Actually I don't think we need `Polymer.com` but we need to default to `document.body` if `document.activeElement` is null , e.g.
`(document.activeElement || document.body).blur();`
See https://github.com/PolymerElements/iron-overlay-behavior/blob/086b930c548dea9a1ce777a154d859cb19e9df6e/iron-overlay-manager.html#L73-L76